Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Coverage"
 uuid = "a2441757-f6aa-5fb2-8edb-039e3f45d037"
 authors = ["Iain Dunning <iaindunning@gmail.com>"]
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
Needs to be a minor bump if we drop Julia 0.7 from compat

Reply:
Good catch.